### PR TITLE
feat: add lateral bar for logged in user

### PR DESCRIPTION
**Pull request by wrongbyte**
adding this 
image

### CODE REVIEW

File: src/components/feed/LateralBar.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
+import '../../styles/global.css';
+
+// TODO: ORGANIZE THIS MESS
```
Comment (noghartt) [inline src/components/feed/LateralBar.tsx:3]: KEK

File: src/components/feed/LateralBar.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+export default function LateralBar() {
+  return (
+    <aside className="lateralbar-wrapper">
+      <div className="lateralbar-icons">
+        <img className="twitter-icon-lateralbar" src="twitter-xxl.png"></img>
+
+        {/* HOME */}
+        <svg viewBox="0 0 24 24" aria-hidden="true" className="svg-lateral">
```
Comment (noghartt) [inline src/components/feed/LateralBar.tsx:12]: I think that you could put those SVGs into another component and just import it here.

And why not [react-icons](https://react-icons.github.io/react-icons/)?

File: src/components/feed/LateralBar.tsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+// TODO: ORGANIZE THIS MESS
+
+export default function LateralBar() {
+  return (
+    <aside className="lateralbar-wrapper">
+      <div className="lateralbar-icons">
+        <img className="twitter-icon-lateralbar" src="twitter-xxl.png"></img>
+
+        {/* HOME */}
```
Comment (noghartt) [inline src/components/feed/LateralBar.tsx:11]: Moving it to another component, isn't necessary to put these comments.

File: src/styles/home.css [hunk 4]
```diff
@@ -0,0 +1,44 @@
+.twitter-feed-wrapper {
```
Comment (noghartt) [inline src/styles/home.css:1]: Why not CSS-in-JS?

**Review by noghartt — APPROVED** [review]
awesome

**Review by daniloab — APPROVED** [review]

**Pull request merged**